approaching_emergency_vehicle_plugin/src/approaching_emergency_vehicle_plugin_node.cpp

```diff
@@ -163,7 +169,7 @@ namespace approaching_emergency_vehicle_plugin
 
     outgoing_emergency_vehicle_response_pub_ = create_publisher<carma_v2x_msgs::msg::EmergencyVehicleResponse>("outgoing_emergency_vehicle_response", 10);
 
-    upcoming_lane_change_status_pub_ = create_publisher<carma_planning_msgs::msg::UpcomingLaneChangeStatus>("upcoming_lane_change_status", 10);
+    approaching_erv_status_pub_ = create_publisher<carma_msgs::msg::UIInstructions>("approaching_erv_status", 10);
```

Comment:
Does this 10 mean 10HZ?

Reply:
10 refers to the Quality of Service 'History Depth' setting for this ROS Topic. This is the max number of messages that will be kept on the publication queue for any given time. Publisher history depth settings are typically set to 5 or 10 for publishers throughout carma-platform.